Neos.Neos/Resources/Private/Fusion/Backend/History/Root.fusion

```diff
@@ -28,15 +28,15 @@ prototype(Neos.Neos:History.EventRenderer) < prototype(Case) {
 # Node events
 #
 
-prototype(Neos.Neos:History.NodeEventRenderer) < prototype(Array) {
+prototype(Neos.Neos:History.NodeEventRenderer) < prototype(Neos.Fusion:Array) {
```

Comment:
Is there a reason why not to use Join?

```suggestion
prototype(Neos.Neos:History.NodeEventRenderer) < prototype(Neos.Fusion:Join) {
```

Reply:
see https://github.com/neos/neos-development-collection/pull/3497#discussion_r833019619